opentelemetry-semantic-conventions/src/opentelemetry/semconv/_incubating/attributes/messaging_attributes.py

```diff
@@ -26,9 +26,10 @@
 A unique identifier for the client that consumes or produces a message.
 """
 
-# MESSAGING_CLIENT_ID: Final = "messaging.client_id"
-
-# Deprecated: Replaced by `messaging.client.id`.
+# MESSAGING_CLIENT_ID : Final = "messaging.client_id"
+"""
```

Comment:
Surprised this now does not raise a warning from linters :shrug:

Reply:
oh, thanks!